backend/app/rest/api/rest_public.go

```diff
@@ -364,6 +365,31 @@ func (s *public) robotsCtrl(w http.ResponseWriter, r *http.Request) {
 	render.PlainText(w, r, "User-agent: *\nDisallow: /auth/\nDisallow: /api/\n"+strings.Join(allowed, "\n")+"\n")
 }
 
+// GET /qr - generates QR for URL in provided body, used for generating telegram link QR. Provided text should start with https://t.me/.
+func (s *public) qrCtrl(w http.ResponseWriter, r *http.Request) {
+	text := r.URL.Query().Get("text")
```

Comment:
Maybe we make sure this method's body is match the the method's name? I mean it called generically `qrCtrl` but in fact this is `qrTelegramCtrl` 

Another thing to consider - maybe we don't really need such a specific method and rather  keep qrCtrl to handle any urls? And maybe renaming "text" query param to "url" will make the api better match to what it does?